android/app/build.gradle

```diff
@@ -7,9 +7,9 @@ if (localPropertiesFile.exists()) {
 }
 
 def flutterRoot = localProperties.getProperty('flutter.sdk')
-if (flutterRoot == null) {
-    throw new GradleException("Flutter SDK not found. Define location with flutter.sdk in the local.properties file.")
-}
```

Comment:
Why?

Reply:
I was having issues with this, my environment couldn't find the "GradleException" class. I can remove this from the PR if it works on your end fine